go.mod

```diff
@@ -2,10 +2,11 @@ module local/build
 
 go 1.17
 
+require github.com/hazelcast/hazelcast-go-client v1.1.0
```

Comment:
Could you change the dependency to v1.3.0 ?

Reply:
Also, why is `go.mod` different from other code sample PRs?

Reply:
Fixed at https://github.com/hazelcast/hazelcast-cloud-go-sample-client/pull/11/commits/785591b859cf7cf5aeece91b341e6b640d4d688f.